module/VuFind/src/VuFind/Search/Favorites/Results.php

```diff
@@ -185,28 +170,25 @@ protected function performSearch()
     {
         $list = $this->getListObject();
         $auth = $this->getAuthorizationService();
-        $this->user = $auth ? $auth->getIdentity() : false;
+        $this->user = $auth ? $auth->getIdentity() : null;
```

Comment:
Could be shortened to `$this->user = $auth?->getIdentity();`

Reply:
Done!